lms/static/sass/views/_login-register.scss

```diff
@@ -415,10 +435,6 @@ $sm-btn-linkedin: #0077b5;
                 border: 1px solid #A5382B;
                 color: white;
             }
-
-            &:hover {
-                box-shadow: 0 2px 1px 0 #8D3024;
-            }
```

Comment:
What's the reason for removing these? So you didn't have to add `box-shadow-color: ...` to the inline styles? I think it's actually a nice change but it won't be my call.